xml/net/protocol.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <protocol>
     <enum name="PacketFamily" type="byte">
+        <comment>Part of the unique ID that identifies each packet</comment>
```

Comment:
Might be better to describe this and action differently like family is the area of the game you're interacting with and action is the operation within that area.

I can't think of a nice comment for it though :P

Reply:
I'm going to lean into the official "OP" and AC" names a bit and describe them as operations and actions in the comments.